mlc_llm/relax_model/llama.py

```diff
@@ -580,13 +584,22 @@ def create_encoding_func(bb: relax.BlockBuilder, config: LlamaConfig) -> None:
                 all_seq_len_shape,
                 past_key_values,
             ] + model.parameters()
+
+            named_params = named_parameters(model)
+            for i, (name, param) in enumerate(list(named_params.items())[:-2]):
+                pidx2pname[i] = name
+                assert param.same_as(params[i + 3])
+                assert not name.startswith("sin") and not name.startswith("cos")
```

Comment:
Can we let the user customize the filter list (not urgent though)?

Reply:
For the LLaMA model implementation we now have, it’s not likely that the weight order can be customized?

Reply:
Okay it's an `assert`, just ignore what I said :)